changelog/unreleased/enhancement-icon-and-image-accessibility

```diff
@@ -0,0 +1,3 @@
+Enhancement: Add properties to make icons and images a11y-compliant
```

Comment:
Is this still an enhancement or possibly a breaking change with all the new stuff around icons/images? We'll release a new major version around the theming anyways so just asking for clarification

Reply:
good catch, it's a breaking change 👍

Reply:
or let's put it like this: it's a breaking change regarding accessibility, not functionality.

Reply:
You're totally right, will change this 🙂